tests/system/providers/google/cloud/cloud_build/example_cloud_build.py

```diff
@@ -42,88 +42,65 @@
+from airflow.utils.trigger_rule import TriggerRule
 
-START_DATE = datetime(2021, 1, 1)
+ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
+PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT")
 
-GCP_PROJECT_ID = os.environ.get("GCP_PROJECT_ID", "aitflow-test-project")
+DAG_ID = "example_gcp_cloud_build"
 
-GCP_SOURCE_ARCHIVE_URL = os.environ.get("GCP_CLOUD_BUILD_ARCHIVE_URL", "gs://airflow-test-bucket/file.tar.gz")
-GCP_SOURCE_REPOSITORY_NAME = os.environ.get("GCP_CLOUD_BUILD_REPOSITORY_NAME", "airflow-test-repo")
+BUCKET_NAME_SRC = f"bucket-src-{DAG_ID}-{ENV_ID}"
+
+GCP_SOURCE_ARCHIVE_URL = os.environ.get("GCP_CLOUD_BUILD_ARCHIVE_URL", f"gs://{BUCKET_NAME_SRC}/file.tar.gz")
+GCP_SOURCE_REPOSITORY_NAME = "test-cloud-build-repository"
 
 GCP_SOURCE_ARCHIVE_URL_PARTS = urlparse(GCP_SOURCE_ARCHIVE_URL)
 GCP_SOURCE_BUCKET_NAME = GCP_SOURCE_ARCHIVE_URL_PARTS.netloc
 
 CURRENT_FOLDER = Path(__file__).parent
-
-# [START howto_operator_gcp_create_build_trigger_body]
-create_build_trigger_body = {
-    "name": "test-cloud-build-trigger",
-    "trigger_template": {
-        "project_id": GCP_PROJECT_ID,
-        "repo_name": GCP_SOURCE_REPOSITORY_NAME,
-        "branch_name": "master",
-    },
-    "filename": "cloudbuild.yaml",
-}
-# [END howto_operator_gcp_create_build_trigger_body]
-
-update_build_trigger_body = {
-    "name": "test-cloud-build-trigger",
-    "trigger_template": {
-        "project_id": GCP_PROJECT_ID,
-        "repo_name": GCP_SOURCE_REPOSITORY_NAME,
-        "branch_name": "dev",
-    },
-    "filename": "cloudbuild.yaml",
-}
+FILE_LOCAL_PATH = str(Path(CURRENT_FOLDER) / "resources" / "file.tar.gz")
 
 # [START howto_operator_gcp_create_build_from_storage_body]
 create_build_from_storage_body = {
     "source": {"storage_source": GCP_SOURCE_ARCHIVE_URL},
-    "steps": [
-        {
-            "name": "gcr.io/cloud-builders/docker",
-            "args": ["build", "-t", f"gcr.io/$PROJECT_ID/{GCP_SOURCE_BUCKET_NAME}", "."],
-        }
-    ],
-    "images": [f"gcr.io/$PROJECT_ID/{GCP_SOURCE_BUCKET_NAME}"],
+    "steps": [{"name": "ubuntu", "args": ['echo', 'Hello world']}],
 }
 # [END howto_operator_gcp_create_build_from_storage_body]
 
 # [START howto_operator_create_build_from_repo_body]
 create_build_from_repo_body: Dict[str, Any] = {
-    "source": {"repo_source": {"repo_name": GCP_SOURCE_REPOSITORY_NAME, "branch_name": "main"}},
-    "steps": [
-        {
-            "name": "gcr.io/cloud-builders/docker",
-            "args": ["build", "-t", "gcr.io/$PROJECT_ID/$REPO_NAME", "."],
-        }
-    ],
-    "images": ["gcr.io/$PROJECT_ID/$REPO_NAME"],
+    "source": {"repo_source": {"repo_name": GCP_SOURCE_REPOSITORY_NAME, "branch_name": "master"}},
+    "steps": [{"name": "ubuntu", "args": ['echo', 'Hello world']}],
 }
 # [END howto_operator_create_build_from_repo_body]
 
 
 with models.DAG(
-    "example_gcp_cloud_build",
-    start_date=START_DATE,
+    DAG_ID,
+    schedule_interval='@once',
```

Comment:
```suggestion
    schedule='@once',
```
Updating to the new `schedule` param. See #25410 and #25648.